server/src/internalClusterTest/java/org/elasticsearch/search/profile/aggregation/AggregationProfilerIT.java

```diff
@@ -224,12 +224,7 @@ private void assertRemapTermsDebugInfo(ProfileResult termsAggResult) {
         assertThat(termsAggResult.getDebugInfo(), hasEntry(COLLECTION_STRAT, "remap using many bucket ords"));
         assertThat(termsAggResult.getDebugInfo(), hasEntry(RESULT_STRAT, "terms"));
         assertThat(termsAggResult.getDebugInfo(), hasEntry(HAS_FILTER, false));
-        // TODO we only index single valued docs but the ordinals ends up with multi valued sometimes
-        assertThat(
-            termsAggResult.getDebugInfo().toString(),
-            (int) termsAggResult.getDebugInfo().get(SEGMENTS_WITH_SINGLE) + (int) termsAggResult.getDebugInfo().get(SEGMENTS_WITH_MULTI),
-            greaterThan(0)
-        );
+        assertThat(termsAggResult.getDebugInfo().toString(), (int) termsAggResult.getDebugInfo().get(SEGMENTS_WITH_SINGLE), greaterThan(0));
```

Comment:
I bumped into this while double checking that I hadn't broken this test. This was fixed in #71241.